jsyntrax/src/main/java/org/atpfivt/jsyntrax/generators/SVGCanvas.java

```diff
@@ -67,6 +67,13 @@ public void scaleByTag(String tag, double scale) {
         }
     }
 
+    public String getCanvasTag() {
```

Comment:
Method is declared but not used and thus it should be deleted.

And by the way: if it _was_ used, I would have asked you to rewrite it using functional style Java for more brevity and clarity

Reply:
https://github.com/atp-mipt/jsyntrax/blob/add_title/jsyntrax/src/main/java/org/atpfivt/jsyntrax/generators/SVGCanvasBuilder.java#L70
Used here